wandb/sdk/interface/interface.py

```diff
@@ -16,6 +16,8 @@
 from typing import Any, Iterable, Optional, Tuple, Union
 from typing import TYPE_CHECKING
 
+import six
```

Comment:
why do you need `six` we are only using py3+ ... `six` to allow support for py2... probably good idea to not introduce it in new code

Reply:
deleted in next commit